### PR TITLE
fix(deps): remove security vulnerabilities

### PR DESCRIPTION
**Pull request by dwojno**
* removed "build" package from dev dependecnies (not found any usage)
* upgraded web3-utils using npm audit tool
* removed yarn.lock file and yarn usage in the lib
  - lib contained 2 lock files which were out of sync with each other
    and with current state of dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "demo8": "node dist/js/demo/8_demo_verifier.js",
     "debug": "node --inspect-brk dist/js/index.js",
     "prebuild": "rm -rf dist/js",
-    "build": "yarn compile && yarn build:typechain && tsc",
+    "build": "npm run compile && npm run build:typechain && tsc",
```
Comment (jrhender) [inline package.json:20]: I don't have much of an opinion, but why the change from yarn to npm?

Comment (dwojno) [reply package.json:20]: in the repo, someone was using both, so I've decided to go with npm as it is already included in node :)
that's the only reason :)

**Review by jrhender — APPROVED** [review]

**Review by michalziobro — APPROVED** [review]
